docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[Kubernetes-dashboard]]
+= The Kubernetes dashboard: Linux session data from your Kubernetes clusters
+
+This page explains how to deploy a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[daemonset] to collect session data from your Kubernetes clusters using the Endpoint and Cloud Security integration, and how to view that data in Kibana’s Kubernetes dashboard.
+
+**Prerequisites**: This feature requires Elastic Stack version 8.4 or higher. Your Elastic deployment must have the Endpoint and Cloud Security integration enabled, and you need an active Fleet Server.
+
+**Support Matrix**: This feature is currently available on GKE and EKS using Linux hosts and Kubernetes versions that match the following specifications:
+|=====================
+| | **Kubernetes versions** | **Container runtime** | **Nose OSes**
+|**EKS**| 1.18; 1.19; 1.20; 1.21| Docker | Amazon Linux 2, Bottlerocket OS
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+You first need to download the daemonset manifest .yaml, then modify it to include your Fleet URL and Enrollment Token before you deploy it to the clusters you want to monitor. After that, your data will appear in the Kubernetes dashboard.
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+1. Download the daemonset manifest http://github.com/elastic/placeholder[here] for GKE and http://github.com/elastic/placeholder[here] for EKS.
+2. In the manifest, fill in the `FLEET_URL` field with your Fleet URL. If you’re not sure how to find it, refer to https://www.elastic.co/guide/en/fleet/current/fleet-settings.html[Fleet UI settings].
```

Comment:
This is helpful to me, but I think the user might get slightly confused with the onboarding here. The way I usually onboard is that, yes I make sure that a fleet server is running. the link walks how to check for that. But once that is done I go through the Endpoint and Cloud Security onboarding workflow. e.g 
1. Home
2. Add integration (call to action button)
3. "Endpoint and Cloud Security" tile
4. Add Endpoint and Cloud Security (call to action button)
5. Go through step 1, step 2 for configure integration and agent policy and click "save and Continue
6. Add Elastic Agent to your host (call to action button)

Here is where i take note of the `--url=` and the token `--enrollment-token=` values which I will plugin to the `FLEET_URL` and `FLEET_ENROLLMENT_TOKEN` respectively within the YAML file 
8.

Reply:
Thanks for this Quan. I agree that these steps should more directly tell users how to find the Fleet URL and Enrollment token, and I've merged some changes in response to this feedback:
- Added explicit indications of where to find those two values (so users won't have to follow any links to find them).
- Added links in the prerequisites section at the top of the doc for how to set up a fleet server and how to deploy the Endpoint and Cloud Security integration, in case users haven't done that yet.

Please let me know if you'd like to see additional changes.